src/components/Navbar/Navbar/style.scss

```diff
@@ -0,0 +1,150 @@
+@import '../../../scss/variables';
+@import '../../../scss/base';
+
+$white: #ffffff;
+*,
+::before,
+::after {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+//Main header to be used for sticky if needed
+.container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background: $white;
+    width: 100%;
+    position: sticky;
+    top:0px;
+    max-width: 100%;
+    z-index: 10;
+
+    .NavbarItems {
+        display: flex;
+        width: 80%;
+        height: 90px;
+        justify-content: center;
+        align-items: center;
+        font-size: 1.2rem;
+    }
+}
+
+.nav-menu {
+    display: grid;
+    grid-template-columns: repeat(8, auto);
+    grid-gap: 10px;
+    list-style: none;
+    text-align: center;
+    width: 80vw;
+    justify-content: center;
+    margin-right: 2rem;
+}
+.nav-links {
+    color: #000;
+    text-decoration: none;
+    padding: 0.5rem 1rem;
+
+    &:hover {
+     color: $primary-green-light;
+     cursor: pointer;
+    }
+}
+
+// Mobile
+.fa-bars {
+    color: $primary-green-dark;
+}
+.nav-links-mobile {
+    display: none;
+}
+.menu-icon {
+    display: none
+}
+
+//Logo To be changed at later time
+// .navbar-logo {
+//     color: #000;
+//     justify-self: start;
+//     margin-left: 20px;
+//     cursor: pointer;
+// }
+.logo {
+    padding-top: 22px;
+    width: 180px;
+    margin-bottom: 18px;
+}
+
+
+//Media
+@media screen and (max-width: 1200px) {
+    
+.NavbarItems {
```

Comment:
Please change this to kebab case to match the rest of your naming conventions something like navbar-items or nav-items